docs/notebooks/utils/downloading_brains.ipynb

```diff
@@ -1,31 +1,19 @@
 {
```

Comment:
It appears as if this notebook had an error when running. We might decide to not push notebooks that failed to run, thoughts?

Reply:
For now, clear notebook output before you push.
In the future we will do this automatically via https://github.com/neurodata/brainlit/issues/84.